components/admin/lmod/SPECS/lmod.spec

```diff
@@ -97,7 +97,7 @@ make DESTDIR=$RPM_BUILD_ROOT install
 ########################################################################
 
 # NOOP if running under known resource manager
-if [ ! -z "\$SLURM_NODELIST" ] | [ ! -z "\$PBS_NODEFILE" ]; then
+if [ -n "\$SLURM_NODELIST" ] || [ -n "\$PBS_NODEFILE" ]; then
```

Comment:
Shouldn't `||` be `&&` ? I.e. if neither is defined then return early.

Reply:
Hmm, good question. Let me take a look.

Reply:
Just tested a couple of combinations and it works as expected. The goal is, if one is set it should return.